packages/lodestar-params/src/utils.ts

```diff
@@ -1,29 +1,90 @@
 import {FAILSAFE_SCHEMA, Schema, Type} from "js-yaml";
-import {TypeMap, typeMap} from "./types";
 import {IBeaconParams} from "./interface";
-import * as constants from "./constants";
+import {ContainerType, BigIntUintType, NumberUintType, BitVectorType, BitListType, Json} from "@chainsafe/ssz";
 
-export function createIBeaconParams(params: Record<string, unknown>): Partial<IBeaconParams> {
-  return convertTypes(params, typeMap);
-}
+const beaconParamsType = new ContainerType({
```

Comment:
```suggestion
export const BeaconParams = new ContainerType<IBeaconParams>({
```
Can you also export this type from src/index.ts?

Reply:
Can you move the ssz type to its own file? eg: src/sszType.ts